### PR TITLE
Yeti YUSD stability Pool

### PR DESCRIPTION
**Pull request by 0xAces**
add YUSD Stability Pool yield

### CODE REVIEW

**Comment by vrtnd** [comment]
Tests error

**Comment by llamatester** [comment]
Error while running yeti-finance adapter: 
        
 
 Test Suites: 1 failed, 1 total
    Tests:       5 failed, 100 passed, 105 total
    Snapshots:   0 total
    Time:        0.507 s
    Ran all test suites.
    
    Nb of pools: 17
     
    
    Sample pools:
    ┌─────────┬──────────────────────────┬─────────────┬────────────────┬─────────────┬────────────────────┬────────────────────┬──────────────────────────────────────────────────┐
    │ (index) │           pool           │    chain    │    project     │   symbol    │       tvlUsd       │        apy         │                 underlyingTokens                 │
    ├─────────┼──────────────────────────┼─────────────┼────────────────┼─────────────┼────────────────────┼────────────────────┼──────────────────────────────────────────────────┤
    │    0    │    'Yeti aUSDC Vault'    │ 'Avalanche' │ 'yeti-finance' │   'aUSDC'   │ 26183592.949004564 │ 1.5712339591844149 │ [ '0x625E7708f30cA75bfd92586e17077590C60eb4cD' ] │
    │    1    │   'Yeti qiUSDCn Vault'   │ 'Avalanche' │ 'yeti-finance' │  'qiUSDCn'  │ 8923594.431400603  │ 1.7500102537005446 │ [ '0xB715808a78F6041E46d61Cb123C9B4A27056AE9C' ] │
    │    2    │    'Yeti aUSDT Vault'    │ 'Avalanche' │ 'yeti-finance' │   'aUSDT'   │  8914843.53489823  │ 3.1449330112260476 │ [ '0x6ab707Aca953eDAeFBc4fD23bA73294241490620' ] │
    │    3    │   'Yeti qiUSDC Vault'    │ 'Avalanche' │ 'yeti-finance' │  'qiUSDC'   │ 544811.5883648744  │ 2.1231750991951337 │ [ '0xBEb5d47A3f720Ec0a390d04b4d41ED7d9688bC7F' ] │
    │    4    │    'Yeti qiETH Vault'    │ 'Avalanche' │ 'yeti-finance' │   'qiETH'   │ 529776.9660794532  │ 1.040100075177397  │ [ '0x334AD834Cd4481BB02d09615E7c11a00579A7909' ] │
    │    5    │    'Yeti qiDAI Vault'    │ 'Avalanche' │ 'yeti-finance' │   'qiDAI'   │ 456473.7486337787  │ 3.0442146265403203 │ [ '0x835866d37AFB8CB8F8334dCCdaf66cf01832Ff5D' ] │
    │    6    │    'Yeti sJOE Vault'     │ 'Avalanche' │ 'yeti-finance' │   'sJOE'    │ 398086.40822986234 │ 21.53861129792798  │ [ '0x6e84a6216eA6dACC71eE8E6b0a5B7322EEbC0fDd' ] │
    │    7    │    'Yeti sAVAX Vault'    │ 'Avalanche' │ 'yeti-finance' │   'sAVAX'   │ 314376.91407810105 │        5.76        │ [ '0x2b2C81e08f1Af8835a78Bb2A90AE924ACE0eA4bE' ] │
    │    8    │ 'Yeti AVAXUSDCJLP Vault' │ 'Avalanche' │ 'yeti-finance' │ 'AVAX-USDC' │ 99371.84551746189  │ 9.060394860950929  │ [ '0xf4003F4efBE8691B60249E6afbD307aBE7758adb' ] │
    │    9    │    'Yeti aDAI Vault'     │ 'Avalanche' │ 'yeti-finance' │   'aDAI'    │ 77571.01527870685  │ 2.152968307033465  │ [ '0x82E64f49Ed5EC1bC6e43DAD4FC8Af9bb3A2312EE' ] │
    └─────────┴──────────────────────────┴─────────────┴────────────────┴─────────────┴────────────────────┴────────────────────┴──────────────────────────────────────────────────┘
    }

File: src/adaptors/yeti-finance/index.js [hunk 1]
```diff
@@ -38,11 +40,11 @@ async function apr() {
 
     const vaultsToExclude = ["WAVAX", "USDC", "WETH", "WBTC", "av3CRV"]
 
-    return [
+    const vaultAPRs = [
         ...vaults.filter(v => v !== "0x0000000000000000000000000000000000000000" && !vaultsToExclude.includes(vaultToName[v]))
     ].map( v => (
         {
-        pool: `Yeti-${vaultToName[v]}-Vault`,
+        pool: `Yeti ${vaultToName[v]} Vault`,
```
Comment (vrtnd) [inline src/adaptors/yeti-finance/index.js]: Please revert this change, this is id field not name

File: src/adaptors/yeti-finance/index.js [hunk 2]
```diff
@@ -52,6 +54,20 @@ async function apr() {
         apy: Number(aprs[vaultToName[v]].APY.value) * 100,
         underlyingTokens: [underlying[v]]
     }));
+
+    const stabilityPool = [{
```
Comment (vrtnd) [inline src/adaptors/yeti-finance/index.js:58]: use object or `array.concat(stabilityPool)`

**Comment by llamatester** [comment]
The yeti-finance adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       106 passed, 106 total
    Snapshots:   0 total
    Time:        0.444 s
    Ran all test suites.
    
    Nb of pools: 17
     
    
    Sample pools:
    ┌─────────┬───────────────────────────┬─────────────┬────────────────┬─────────────┬────────────────────┬────────────────────┬──────────────────────────────────────────────────┐
    │ (index) │           pool            │    chain    │    project     │   symbol    │       tvlUsd       │        apy         │                 underlyingTokens                 │
    ├─────────┼───────────────────────────┼─────────────┼────────────────┼─────────────┼────────────────────┼────────────────────┼──────────────────────────────────────────────────┤
    │    0    │    'Yeti-aUSDC-Vault'     │ 'Avalanche' │ 'yeti-finance' │   'aUSDC'   │ 26157693.242929038 │ 1.5712339591844149 │ [ '0x625E7708f30cA75bfd92586e17077590C60eb4cD' ] │
    │    1    │ 'Yeti-YUSD-StabilityPool' │ 'Avalanche' │ 'yeti-finance' │   'YUSD'    │ 15368921.03223435  │ 3.0846092075674294 │ [ '0x111111111111ed1D73f860F57b2798b683f2d325' ] │
    │    2    │   'Yeti-qiUSDCn-Vault'    │ 'Avalanche' │ 'yeti-finance' │  'qiUSDCn'  │  8923225.03654418  │ 1.7500102537005446 │ [ '0xB715808a78F6041E46d61Cb123C9B4A27056AE9C' ] │
    │    3    │    'Yeti-aUSDT-Vault'     │ 'Avalanche' │ 'yeti-finance' │   'aUSDT'   │ 8915629.582306325  │ 3.1449330112260476 │ [ '0x6ab707Aca953eDAeFBc4fD23bA73294241490620' ] │
    │    4    │    'Yeti-qiUSDC-Vault'    │ 'Avalanche' │ 'yeti-finance' │  'qiUSDC'   │ 544784.7198336021  │ 2.1231750991951337 │ [ '0xBEb5d47A3f720Ec0a390d04b4d41ED7d9688bC7F' ] │
    │    5    │    'Yeti-qiETH-Vault'     │ 'Avalanche' │ 'yeti-finance' │   'qiETH'   │ 538942.8428927072  │ 1.040100075177397  │ [ '0x334AD834Cd4481BB02d09615E7c11a00579A7909' ] │
    │    6    │    'Yeti-qiDAI-Vault'     │ 'Avalanche' │ 'yeti-finance' │   'qiDAI'   │ 456729.89938001364 │ 3.0442146265403203 │ [ '0x835866d37AFB8CB8F8334dCCdaf66cf01832Ff5D' ] │
    │    7    │     'Yeti-sJOE-Vault'     │ 'Avalanche' │ 'yeti-finance' │   'sJOE'    │ 404123.6971043254  │ 21.53861129792798  │ [ '0x6e84a6216eA6dACC71eE8E6b0a5B7322EEbC0fDd' ] │
    │    8    │    'Yeti-sAVAX-Vault'     │ 'Avalanche' │ 'yeti-finance' │   'sAVAX'   │ 320865.6918339586  │        5.76        │ [ '0x2b2C81e08f1Af8835a78Bb2A90AE924ACE0eA4bE' ] │
    │    9    │ 'Yeti-AVAXUSDCJLP-Vault'  │ 'Avalanche' │ 'yeti-finance' │ 'AVAX-USDC' │ 100344.71656862138 │ 9.060394860950929  │ [ '0xf4003F4efBE8691B60249E6afbD307aBE7758adb' ] │
    └─────────┴───────────────────────────┴─────────────┴────────────────┴─────────────┴────────────────────┴────────────────────┴──────────────────────────────────────────────────┘
